db/schema.rb

```diff
@@ -426,7 +426,38 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
+  add_foreign_key "allergy_intolerances", "profiles"
```

Comment:
This will do nothing as this file is generated from the db migrations.   All database modifications need to happen through migrations, any mods in here will eventually be lost

Reply:
That's what the migration is for, this is just the updated schema file after the migration has been run.